jazkarta/shop/interfaces.py

```diff
@@ -65,10 +72,21 @@ class IProduct(model.Schema):
         required=False,
     )
 
+    related_products = RelationList(
+        title='Related products',
+        description=u'Recommendations to users who bought this product, shown during checkout.',
+        default=[],
+        required=False,
+        value_type=schema.Choice(
+            source=CatalogSource(
+                object_provides='jazkarta.shop.interfaces.IProduct'),
+        )
+    )
+
```

Comment:
Like on my comment above, I was thinking this field should be named "Recommended products".